src/libs/vehicle/ardupilot/defaults.ts

```diff
@@ -0,0 +1,51 @@
+import { MAVLinkType } from '@/libs/connection/m2r/messages/mavlink2rest-enum'
+
+/**
+ * The default frequency in Hertz used for each message type.
+ * When initially connecting to the vehicle, the GCS will usually request each of these messages at the specified frequency.
+ */
+export const defaultMessageFrequency = {
+  [MAVLinkType.RAW_IMU]: 2,
+  [MAVLinkType.SCALED_IMU2]: 2,
+  [MAVLinkType.SCALED_IMU3]: 2,
+  [MAVLinkType.SCALED_PRESSURE]: 2,
+  [MAVLinkType.SCALED_PRESSURE2]: 2,
+  [MAVLinkType.SCALED_PRESSURE3]: 2,
+  [MAVLinkType.SYS_STATUS]: 2,
+  [MAVLinkType.POWER_STATUS]: 2,
+  [MAVLinkType.MOUNT_STATUS]: 2,
+  [MAVLinkType.MEMINFO]: 2,
+  [MAVLinkType.GPS_RAW_INT]: 2,
+  [MAVLinkType.GPS_RTK]: 2,
+  [MAVLinkType.GPS2_RAW]: 2,
+  [MAVLinkType.GPS2_RTK]: 2,
+  [MAVLinkType.NAV_CONTROLLER_OUTPUT]: 2,
+  [MAVLinkType.FENCE_STATUS]: 2,
+  [MAVLinkType.NAMED_VALUE_FLOAT]: 2,
+  [MAVLinkType.GLOBAL_POSITION_INT]: 3,
+  [MAVLinkType.SERVO_OUTPUT_RAW]: 2,
+  [MAVLinkType.RC_CHANNELS]: 2,
+  [MAVLinkType.RC_CHANNELS_RAW]: 2,
+  [MAVLinkType.ATTITUDE]: 10,
+  [MAVLinkType.SIMSTATE]: 10,
+  [MAVLinkType.AHRS2]: 10,
+  [MAVLinkType.PID_TUNING]: 10,
+  [MAVLinkType.VFR_HUD]: 10,
```

Comment:
I would say that 16Hz would be ideal for us to have an interface that looks fluid. The same apply to the others 10Hz, that are probably used for UI widgets.

Reply:
Agree.